source/javascripts/application.js.coffee

```diff
@@ -4,3 +4,11 @@
 $ ->
   $('#main_nav').stickyNavigation()
   $('#testimonials').testimonialSlides()
+
```

Comment:
I'd rather prefer this to be included in a reusable plugin or whatever as the sticky navigation as it's also repeated in the blog.